### ISSUE TITLE
Testing separate_fields_by_distance is not actually testing anything

### ISSUE THREAD

**Issue by dragly** [CINPLA/spatial-spike-fields#6]
The following lines compares `pos_true` to `pos_true`, which will always be true. It should test against `peaks`, but changing `pos_true` to `peaks` fails the test currently, so there is something else that needs to be done:

https://github.com/CINPLA/spatial-maps/blob/18c4281446d898b11daa15a4395ea130e0b87dfe/spatial_maps/tests/test_gridcells.py#L48-L52

**Comment by lepmik** [comment]
good catch!! should be `peaks`, probably just some shape difference in ordering of the peaks or similar. I'll fix soon
